### PR TITLE
Create Minimum_cost_path.py

### PR DESCRIPTION
**Pull request by ManishKumar219**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: dynamic_programming/Minimum_cost_path.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
```
Comment (CaedenPH) [inline dynamic_programming/Minimum_cost_path.py:1]:
```suggestion
```
The code in this repository is targeted towards python 3.10+

File: dynamic_programming/Minimum_cost_path.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def minimum_cost_path(matrix: list[list[int]]) -> int:
+    """
+    Find the minimum cost traced by all possible paths from top left to bottom right in
+    a given matrix
+
+    >>> minimum_cost_path([[2, 1], [3, 1], [4, 2]])
+    6
+
```
Comment (CaedenPH) [inline dynamic_programming/Minimum_cost_path.py:11]:
```suggestion
```

File: dynamic_programming/Minimum_cost_path.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from __future__ import annotations
+
+
+def minimum_cost_path(matrix: list[list[int]]) -> int:
+    """
+    Find the minimum cost traced by all possible paths from top left to bottom right in
```
Comment (realstealthninja) [inline dynamic_programming/Minimum_cost_path.py:6]: Adding a few more doctests of cases in which paths are negative integers would be nice

**Comment by cclauss** [comment]
Duplicate https://github.com/TheAlgorithms/Python/blob/master/dynamic_programming/minimum_cost_path.py

**Pull request closed without merging**